src/components/ZipCodeInput.js

```diff
@@ -1,15 +1,19 @@
-import { useEffect } from "react"
-
+import { useEffect, useState } from "react"
 import { Button, Stack, TextField } from "@mui/material"
 import SearchIcon from "@mui/icons-material/Search"
 import fetchPetList from "../components/fetchPetList"
 import { useContext } from "react"
 import { PetsContext } from "../context"
+import { LoadingContext } from "../context"
 import { SearchContext } from "../context"
 
 const ZipCodeInput = () => {
   //using PetsContext and SearchContext
   const { setPetList } = useContext(PetsContext)
+  const { setIsLoading } = useContext(LoadingContext)
+  const [zipcodeFormValues, setZipcodeFormValues] = useState("")
+
```

Comment:
These lines can be deleted. I removed them when I created the <SearchContext/> which takes care of the zipcode value now.